### PR TITLE
Emphasize exceptions to culture name two-letter rule

### LINKED ISSUE #12029: Add cultures that do not follow the xx-xx language/country/region convention
@LokiMidgard commented on [Thu Apr 25 2019](https://github.com/dotnet/docs.de-de/issues/138)

>When you package your application's resources, you must name them using the resource naming conventions that the common language runtime expects. The runtime identifies a resource by its culture name. Each culture is given a unique name, which is usually a combination of a *two-letter, lowercase culture name* associated with a language and, if required, a *two-letter, uppercase subculture name* associated with a country or region. The subculture name follows the culture name, separated by a dash (-). Examples include ja-JP for Japanese as spoken in Japan, en-US for English as spoken in the United States, de-DE for German as spoken in Germany, or de-AT for German as spoken in Austria. See the National Language Support (NLS) API Reference at the Go Global Developer Center for a complete list of culture names.

Generating a Resource using Visual Studio (2019) and setting language to different simplified chinise versions result in following files
 - Form1.zh-Hans.resx
 - Form1.zh-Hans-HK.resx
 - Form1.zh-Hans-MO.resx

The subculture does not consists of a two-letter subculture code.

---
#### Dokumentdetails

⚠ *Bearbeiten Sie diesen Abschnitt nicht. Er ist für die Verknüpfung von docs.microsoft.com zum GitHub-Artikel erforderlich.*

* ID: 4e742bf0-abf8-259c-5194-3687febd38b0
* Version Independent ID: 6d1d9bfb-6174-6215-befe-641babe8a746
* Content: [Verpacken und Bereitstellen von Ressourcen in .NET-Apps](https://docs.microsoft.com/de-de/dotnet/framework/resources/packaging-and-deploying-resources-in-desktop-apps#resource-naming-conventions)
* Content Source: [docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md](https://github.com/dotnet/docs.de-de/blob/live/docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md)
* Product: **dotnet-framework**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #12029

### CODE REVIEW

File: docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md [hunk 1]
```diff
@@ -37,21 +37,21 @@ Applications rely on the .NET Framework Resource Manager, represented by the <xr
...
 
-## Resource Naming Conventions
+## Resource naming conventions
 
 When you package your application's resources, you must name them using the resource naming conventions that the common language runtime expects. The runtime identifies a resource by its culture name. Each culture is given a unique name, which is usually a combination of a two-letter, lowercase culture name associated with a language and, if required, a two-letter, uppercase subculture name associated with a country or region. The subculture name follows the culture name, separated by a dash (-). Examples include ja-JP for Japanese as spoken in Japan, en-US for English as spoken in the United States, de-DE for German as spoken in Germany, or de-AT for German as spoken in Austria. See the **Language tag** column in the [list of language/region names supported by Windows](https://docs.microsoft.com/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c). Culture names follow the standard defined by [BCP 47](https://tools.ietf.org/html/bcp47).
 
+> [!NOTE]
+> There are some exceptions for the two-letter culture names such as `zh-Hans` for Chienese (Simplieied).
```
Comment (pkulikov) [inline docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md]:
```suggestion
> There are some exceptions for the two-letter culture names such as `zh-Hans` for Chienese (Simplifi
ed).
```

Comment (Youssef1313) [reply docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md]: @pkulikov, Thanks!

File: docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md [hunk 2]
```diff
@@ -37,21 +37,21 @@ Applications rely on the .NET Framework Resource Manager, represented by the <xr
...
 
-## Resource Naming Conventions
+## Resource naming conventions
 
 When you package your application's resources, you must name them using the resource naming conventions that the common language runtime expects. The runtime identifies a resource by its culture name. Each culture is given a unique name, which is usually a combination of a two-letter, lowercase culture name associated with a language and, if required, a two-letter, uppercase subculture name associated with a country or region. The subculture name follows the culture name, separated by a dash (-). Examples include ja-JP for Japanese as spoken in Japan, en-US for English as spoken in the United States, de-DE for German as spoken in Germany, or de-AT for German as spoken in Austria. See the **Language tag** column in the [list of language/region names supported by Windows](https://docs.microsoft.com/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c). Culture names follow the standard defined by [BCP 47](https://tools.ietf.org/html/bcp47).
 
+> [!NOTE]
+> There are some exceptions for the two-letter culture names such as `zh-Hans` for Chienese (Simplified).
```
Comment (rpetrusha) [inline docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md]:
```suggestion
> There are some exceptions for the two-letter culture names, such as `zh-Hans` for Chinese (Simplified).
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding this note, @Youssef1313. I've suggested a change.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional change, @Youssef1313. I'll merge your PR now.

**Pull request merged**